server/src/application/application.controller.ts

```diff
@@ -16,6 +17,11 @@ export class ApplicationController {
     return this.applicationService.getApplication(applicationId);
   }
 
+  @Get()
+  getApplications(@Query() query: GetApplicationsQuery): Promise<Application[]> {
```

Comment:
It should be a DTO!

Reply:
Remade into a DTO similar to save-applications.dto.ts